### ISSUE TITLE
Do not echo `stop-commands` command

### ISSUE THREAD

**Issue by ylemkimon** [actions/runner#807]
**Describe the enhancement**
It is recommended to disable command processing via `stop-commands` when logging untrusted information in https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/. However, the token used to stop commands is echoed and visible in logs.

If the logged information is dynamic, i.e., can be changed after the token is printed to the log, it can re-enable workflow commands.

**Comment by rcowsill** [comment]
To avoid this issue you can add a mask for the resume token:

```Shell
jobs:
  example-job:
    name: example
    runs-on: ubuntu-latest
    steps:
    - name: log untrusted output
      run: |
    steps:
      - run: |
          # Generate a unique resume token for this workflow run
          COMMAND_RESUME_TOKEN=$(echo -n ${{ github.token }} | sha256sum | head -c 64)

          # Mask the resume token's value in the logs
          echo "::add-mask::$COMMAND_RESUME_TOKEN"

          # Stop command workflow processing
          echo "::stop-commands::$COMMAND_RESUME_TOKEN"

          # Log untrusted output
          echo "::notice::Untrusted output"

          # Resume workflow command processing
          echo "::$COMMAND_RESUME_TOKEN::"

          # Log trusted output
          echo "::notice::Trusted output"
```

**Comment by assignUser** [comment]
It looks like the value is masked automatically now; :+1:

**Comment by ylemkimon** [comment]
Fixed via #1371.
